Formula/m/mariadb.rb

```diff
@@ -59,8 +63,6 @@ class Mariadb < Formula
   fails_with gcc: "5"
 
   def install
-    ENV.cxx11
-
```

Comment:
Should be unnecessary - https://github.com/MariaDB/server/blob/mariadb-11.5.2/CMakeLists.txt#L88